WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/RefundableOrderItem.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import Yosemite
+
+/// Groups an order item and its quantity available for refund
+///
+struct RefundableOrderItem {
+    /// Original purchased item
+    ///
+    let item: OrderItem
+
+    /// Current quantity available for refund
+    ///
+    let quantity: Int
+
+    /// Quantity expressed as decimal
+    ///
+    var decimalQuantity: Decimal {
```

Comment:
nit: maybe this derived property could be in an extension so that it's easy to see the main properties?